spec/annotate/annotate_models_spec.rb

```diff
@@ -943,7 +943,8 @@ def self.when_called_with(options = {})
         [:active,     :boolean, { limit: 1,  comment: 'ＡＣＴＩＶＥ' }],
         [:name,       :string,  { limit: 50, comment: 'ＮＡＭＥ' }],
         [:notes,      :text,    { limit: 55, comment: 'ＮＯＴＥＳ' }],
-        [:no_comment, :text,    { limit: 20, comment: nil }]
+        [:no_comment, :text,    { limit: 20, comment: nil }],
+        [:location,   :geometry_collection, { limit: nil, comment: nil }]
```

Comment:
With this new column, this spec will fail with an ArgumentError: negative argument if you comment out the newly-added logic in lib/annotate/annotate_models.rb